### PR TITLE
SEO edits whats-business-voice

### PR DESCRIPTION
**Pull request by jdmartinez36**
SEO edits for the description, lengthening and adding keywords. Added a section per the recommendations to get keywords related to small business into the article. Content is essentially marketing info to emphasize the small business potential.

Made minor edits for MS standards in body.

@DaniHalfin

### CODE REVIEW

**Comment by DaniHalfin** [comment]
I'm reviewing. Thanks.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/business-voice/whats-business-voice.md [hunk 1]
```diff
@@ -30,11 +31,11 @@ Business Voice is right for you if your organization has 300 or fewer people and
...
 - Microsoft 365 Business Standard
 - Microsoft 365 Business Premium
-- Microsoft Apps for business 
-- Microsoft Apps for enterprise
+- Microsoft Apps for Business
+- Microsoft Apps for Enterprise
 - Microsoft 365 F1
 - Office 365 F1
-- Microsoft and Office 365 Enterprise E1, E3 or E5
+- Microsoft 365 and Office 365 Enterprise E1, E3 or E5
```
Comment (JohanFreelancer9) [inline Teams/business-voice/whats-business-voice.md:38]:
```suggestion
- Microsoft 365 and Office 365 Enterprise E1, E3, or E5
```

**Comment by scanum** [comment]
@DaniHalfin Hi! Is this PR going to be closed? Thanks!

**Comment by LolaJacobsen** [comment]
@dstrome will you review the Business Voice blurb at the bottom of this PR please? Dani issued this PR with SEO updates. But I don't feel qualified to TR the BV section. Thanks!

**Pull request merged**